### DISCUSSION TITLE
running air for a new project in windows

### DISCUSSION THREAD

**Discussion by ljmerza** [air-verse/air#438] [Q&A]
hello im trying to get air setup to run on an example project on my windows machine. I installed air via `go get` and i ran `air init` to create a default .air.toml file. When i run `air` command though i get 

```
watching .
!exclude tmp
building...
package github.com/go-chi/chi
        imports github.com/go-chi/chi: import cycle not allowed
failed to build, error: exit status 1
running...
'C:\Users\lmerz\projects\golang\tmp\main.exe' is not recognized as an internal or external command,
operable program or batch file.
```

im nots ure how to fix this. the `go/bin` is in my env path.  my default config is 

```
[build]
  args_bin = []
  bin = "tmp\\main.exe"
  cmd = "go build -o ./tmp/main.exe ."
```

**Comment by Yvo68** [comment]
Had the same issue.
Fixed it by modifying the .air.toml with:
full_bin = "./tmp/main.exe"
